README.md

```diff
@@ -7,26 +7,30 @@
 ---
 
 ### Install build dependencies:
+
 ```
 npm install
 ```
 
 ### Work locally (with source file watching and browser refresh):
+
 ```
 npm start
 ```
 
 ### Build locally (and better simulate what Cloudflare Pages will produce):
+
 ```
 (you'll need [cloudflared](https://github.com/cloudflare/cloudflared) and [ImageMagick](https://imagemagick.org/index.php) installed for some of the build tasks)
 npm run build
 ```
 
```

Comment:
````suggestion

### Format your code

```
prettier -w
```
````

Reply:
Added `prettier -w .` and fixed other formatting issues in the latest commit.